backend/src/backend/main.py

```diff
@@ -137,4 +137,18 @@ async def add_book(book_payload: BookAddSchema):
     return {"msg": "ok", "uuid": str(book.id), "title": book.title.ident}
 
 
+@database.transaction()
+@api.get("/titles/", status_code=200, response_model=TitlesListSendSchema)
```

Comment:
User `/titles` as route path. Will work for both

Reply:
I also think it's time to split views into different modules.

Let's have something like the following for now:

```
routes/
  __init__.py
  books.py
  titles.py
```